tests/foreman/ui/test_errata.py

```diff
@@ -954,11 +948,15 @@ def test_content_host_errata_search_commands(session, module_org, module_repos_c
             result = [item['Name'] for item in result]
             assert clients[1].hostname in result, 'Needs-RHBA host not found'
             # Search for applicable RHSA errata by Errata ID
-            result = session.contenthost.search(f'applicable_errata = {FAKE_2_ERRATA_ID}')
+            result = session.contenthost.search(
+                f'applicable_errata = {settings.repos.yum_6.errata[2]}'
```

Comment:
Want to double check with you @Griffin-Sullivan that its intentional to change these from `FAKE_2_ERRATA_ID` to `yum_6`, instead of `yum_2`?

Reply:
Yes `FAKE_2_ERRATA_ID = 'RHSA-2012:0055'` which is equal to `yum_6.errata[2]`. From what I gathered, the errata IDs listed in constants were not associated with the fake yum repos. Instead the errata IDs were just counting up (FAKE_1, FAKE_2, FAKE_3, etc)